collectors/keywords.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+WHITELIST = ["Spring", "dotnet", "kernel"]
```

Comment:
Are these strings really the only things we're interested in?

Reply:
No, there are also a few `.net` combinations. These sources are from Assembler.